lite2/example_test.go

```diff
@@ -48,17 +52,16 @@ func TestExample_Client_AutoUpdate(t *testing.T) {
 	if err != nil {
 		stdlog.Fatal(err)
 	}
+	trustedStore := dbs.New(db, chainID)
+
+	err = trustedStore.SaveSignedHeaderAndValidatorSet(header, vals)
 
-	c, err := NewClient(
+	c, err := NewNetClientFromTrustedStore(
```

Comment:
better to create another test

Reply:
do you want me to create a separate file for testing the helper funcs then?